rules/autoconsent/hornbach-de.json

```diff
@@ -0,0 +1,7 @@
+{
+  "name": "hornbach-de",
+  "detectCmp": [{ "eval": "!!document.querySelector('#usercentrics-root').shadowRoot.querySelector('#uc-center-container')"}],
```

Comment:
Judging by the class names, it's [usercentrics](https://usercentrics.com/), should we have a more generic solution?

Reply:
👍 nice find,  I should for sure rename it. I'll have to see if we run into more example sites to make a general case.

Reply:
Would it make sense to restrict these rules to specific websites? In my understanding some of them are one-off fixes for these websites, so we shouldn't execute them on other pages at all. I see that there's a `UrlRule` type but I don't think it's used in the CMP logic as of now. Maybe we could add logic to ensure site-specific rules are only executed on their respective domains? That could make rule execution faster for other sites as well.

Reply:
This ended up being a bigger CMP. I moved it over to https://github.com/duckduckgo/autoconsent/pull/3/files#diff-b383aa32dd7c9e721949d46b3424e4076abd6e2eb0b74efae275c6f35630d102. 

I'll make a followup for the UrlRule.